app-text/hunspell/hunspell-1.4.2.recipe

```diff
@@ -73,6 +83,10 @@ INSTALL()
 	make install
 
 	rm $libDir/charset.alias $libDir/*.la
+
+	# remove any existing (but empty) data/locale/*/LC_MESSAGES directory
+	rmdir $dataDir/locale/*/LC_MESSAGES $dataDir/locale/* || true
```

Comment:
I did _not_ dare to also
```
	rm $dataDir/locale/locale.alias
```
Shall I do so, before the **`rmdir`**, and then also **`rmdir $dataDir`** ?
Because the only file in **`$dataDir`** is **`data/locale/locale.alias`**, but it either gets shadowed by the **`locale.alias`** from another package or shadows all **`locale.alias`** from the other packages.

Reminder: on some other packages we already drop **`data/locale/charset.alias`**, so I was wondering if we should also treat this **`locale.alias`** file same way we already treat the **`charset.alias`** file.
Thanks!